decidim-comments/app/services/decidim/comments/new_comment_notification_creator.rb

```diff
@@ -28,14 +31,15 @@ def initialize(comment, mentioned_users)
       # Returns nothing.
       def create
         notify_mentioned_users
+        notify_mentioned_groups if mentioned_groups
```

Comment:
shouldn't the invocation of `notify_mentioned_groups` be equal to `notify_mentioned_users`?
Why not control when mentioned_groups is `nil` inside the method?

Reply:
I think I had a reason for that but forgotten. Will do your suggestion!